src/main/java/org/takes/facets/fork/FkRegex.java

```diff
@@ -76,7 +76,7 @@
  * <p>The class is immutable and thread-safe.
  *
  * @author Yegor Bugayenko (yegor@teamed.io)
- * @version $Id$
+ * @version $Id: f2810e906b1f0c42db9c9f328b2545dad77a18b5 $
```

Comment:
@mkordas Any clue why the id is keep on changing? Is it my intellij configurations making this?